src/components/buttons/ButtonsContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { styled } from '../hoc/styled'
+import { MainButton, MainButtonType } from '../themed/MainButton'
+
+export interface ButtonContainerProps {
+  label: string
+  onPress: () => void | Promise<void>
+  disabled?: boolean
+}
+
+interface Props {
+  buttons: Map<MainButtonType, ButtonContainerProps>
+  layout: 'row' | 'column'
+}
+
+/**
+ * A consistently styled view for displaying button layouts.
+ */
+export const ButtonsContainer = React.memo(({ buttons, layout = 'column' }: Props) => {
```

Comment:
The big question: Can we use this for ButtonsModal, or not? If so, then we should do that. If not, then what do we need to do to get there? Your comment indicates that we want this to be the goal.

Reply:
Yeah that was the motivation - ButtonsModal and a case-by-case basis where we update the parent and ButtonsContainer gradually